projects/angular-rave/src/lib/rave-options.ts

```diff
@@ -51,6 +51,8 @@ export interface RaveOptions {
    * This allows you select the payment option you want for your users, possible values are card, account or both
    */
   payment_method?: string;
+
```

Comment:
Can you add the description from the rave documentation as a comment above here.  You can also add 1 or 2 examples.

`card`, `account`, `ussd`, `card,account`

Reply:
done 🙂